lib/forest_liana/version.rb

```diff
@@ -1,3 +1,3 @@
 module ForestLiana
-  VERSION = "5.2.2"
+  VERSION = "6.4.0-development"
```

Comment:
[MISC] If you're preparing a beta release, shouldn't you put it on another branch than master? it might block further releases if not